### DISCUSSION TITLE
How to change the order of appointments?

### DISCUSSION THREAD

**Discussion by Wunderharke** [FossifyOrg/General-Discussion#472] [Q&A] [answered]
I would like to reorder my calender (some are DAV synced) so that a specific calender's appointments are on top in every view.

Any help appreciated. Thanks!

**Comment by Aga-C** [comment, marked as answer]
Events and tasks are sorted by date, there's no possibility to change that order.

**Reply by Wunderharke** [reply]
Okay thanks for this clear answer. Would love to be able to prioritize a calender for monthly view anyway.

**Reply by Aga-C** [reply]
If you'd like, you can raise a [feature request](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) with a full description of how you see this feature.
